python/sosmurf/SessionManager.py

```diff
@@ -1,6 +1,15 @@
 from spt3g import core
 import yaml
 import time
+from enum import Enum
+
+class FlowControl:
```

Comment:
This isn't actually an Enumeration without the `(Enum)` inheritance.

Reply:
oh yeah I realize that, I guess its a stray import statement from when I tried that. Is there any real benefit to making it an actual Enum?

Reply:
Stray import and docstring I suppose. Enum's do fit the purpose of what we're doing with the FlowControl class. (And to more specifically answer your question, they have features that do things like prevent you from double mapping values, etc.)

I guess I would say since it's defined as an Enum everywhere else we currently use it, is there any benefit to not having it also be an Enum here?